sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -283,6 +288,25 @@ object ScalaNativePluginInternal {
       val result = compileIfChanged(inputFiles)
       binary
     },
+    nativeExternalDependencies := {
+      val forceCompile = (compile in Compile).value
+
+      val classes = classDirectory.target
+      val progDir = VirtualDirectory.real(classes)
+      val prog    = tools.Path(progDir)
+
+      import nir.Shows._
```

Comment:
I think it's best to keep all of the block-local imports in the very beginning of the block whenever possible.